src/main/webapp/dev/js/common/remindModal.js

```diff
@@ -15,6 +15,46 @@ function populateCheckBoxes($button) {
     }
 }
 
+function bindingSelectAllCheckBoxes() {
+    // set default checkboxes as unchecked
+    $('#remindAll').prop('checked', false);
+    $('#remindNotSubmitted').prop('checked', false);
+    // checks binding of select all checkboxes populated in the header
+    $('#remindAll').on('change', function () {
+        if (this.checked) {
+            $('input[class^="student-"]').prop('checked', true);
```

Comment:
Just curious, why don't you just use `student-responded` here and other places in this file?

Reply:
as this checks all box for all the students, by putting class^="student-", it will select all the classes with student-responded and student-not-responded, hence using this!

Reply:
Ah I see 👍 just a reminder, there may be other input classes that share the same pattern and may lead to undesired effects, so always be careful when you use wildcards.